lib/jcsda-emc/spack-stack/stack/cmd/stack_cmds/create.py

```diff
@@ -51,44 +51,80 @@ def container_config_help():
     help_string = "Pre-configured container." + os.linesep
     help_string += "Available options are: " + os.linesep
     for config in container_configs:
+        if config == "README.md":
```

Comment:
Alternatively, we could use the logic from `lib/jcsda-emc/spack-stack/tests/test_stack_create.py`:
```
        _, _, containers = next(os.walk(container_path))
        # Exclude files like "README.md"
        containers = [x for x in containers if x.endswith(".yaml")]
```

Reply:
I think this alternate logic would be better as long as we know that config is always going to be in files with a `.yaml` extension. That way we don't pick up an undesirable file if something besides `README.md` shows up.

Would there be cases of config files using a `.yml` extension, ie is it worthwhile to add the check `x.endswith(".yml")` to the test in the alternate logic.

Reply:
I agree, the alternate logic is better. The spack convention is to always use `.yaml`, I don't think we need `.yml`. (Also, the logic in the create container code assumes `.yaml`, because that's what it appends to the arguments.

Reply:
Done